gradient/api_sdk/models/notebook.py

```diff
@@ -57,6 +58,7 @@ def url(self):
 @attr.s
 class NotebookStart(object):
     notebook_id = attr.ib(type=str, default=None)
+    machine_type = attr.ib(type=str, default=None)
```

Comment:
^

Reply:
I would like to leave the old fields in the models because the responses have not been updated to the new field names yet.